apps/web/src/components/activity/ExecutionDetailsModal.tsx

```diff
@@ -65,12 +65,9 @@ export function ExecutionDetailsModal({
   origin: string;
 }) {
   const theme = useMantineTheme();
-  const {
-    data: response,
-    isLoading,
-    isFetching,
-  } = useQuery(['activity', notificationId], () => getNotification(notificationId), {
+  const { data: response, isLoading } = useQuery(['activity', notificationId], () => getNotification(notificationId), {
     enabled: !!notificationId,
+    refetchInterval: 1000,
```

Comment:
What's this for?

Reply:
So if the modal is opened, it will be updated with the statuses live. 
https://www.loom.com/share/c9681141a167476babd80c77fdffbee3

Reply:
Oh nice. Actually very cool.
Just wondering if 1s is a very small interval for the refetching. Just ponder the number of calls the modal would make against the needs of the user to have that live updated by the moment.

Reply:
Sure, we can definitely make it higher. Good point.